src/components/Popover/index.js

```diff
@@ -11,23 +11,40 @@ import PopoverWithoutOverlay from '../PopoverWithoutOverlay';
  * On small screen widths, it uses BottomDocked modal type, and a Popover type on wide screen widths.
  */
 function Popover(props) {
-    if (!props.fullscreen && !props.isSmallScreenWidth) {
+    const {isVisible, onClose, isSmallScreenWidth, fullscreen, animationInTiming, onLayout, animationOutTiming, disableAnimation, withoutOverlay, anchorPosition} = props;
+    // Not adding this inside the PopoverProvider
```

Comment:
NAB - I'm pretty sure we like to have a blank line above any comments. Not sure why the automations didn't catch that though, so maybe I'm wrong?

Reply:
Ah looks like it might not offer that as an option, so maybe we don't have anything automated for that one. Do you mind terribly adding the empty line? I'm checking to make sure I'm not inventing that rule, but I'm pretty sure it's real, haha